### DISCUSSION TITLE
Transform of X_baseline in qNEHVI (qNoisyExpectedHypervolumeImprovement) and qLNEHVI (qLogNoisyExpectedHypervolumeImprovement) instantiating necessary?

### DISCUSSION THREAD

**Discussion by MoBurmeister** [meta-pytorch/botorch#2478] [Q&A] [answered]
Hello together, 

as mentioned in the title, I was wondering if it is necessary to transform the handed over X_baseline, when initiating the qNEHVI or qLNEHVI acquisition function. 

In this tutorial it is done: https://botorch.org/tutorials/multi_objective_bo

But in most of the discussed topics and shared code here in the discussion forum it is not done. Also when sharing part of my code last time (https://github.com/pytorch/botorch/discussions/2470) it was not pointed out/mentioned in the responses. 

The docstrings of both acquisition functions do not really provide clarity: 

qNEHVI & qLNEHVI:

"X_baseline: A `r x d`-dim Tensor of `r` design points that have already
                been observed. These points are considered as potential approximate
                pareto-optimal design points."

I tried to find out more in the code of botorch, but wasnt really able to get a hint about it.

If it should be done, the transforms of the model should be applied? 
And also if transforms are necessary, this can maybe be added to the docstring for more clarity. 

Thanks in advance for any help :)

**Comment by saitcakmak** [comment, marked as answer]
Hi @MoBurmeister. `X_baseline` should be consistent with the inputs provided to the model. `X_baseline` is typically same as `train_X`, so if you pre-process `train_X` before initializing the model, you should apply the same process to `X_baseline` as well. This is the path the tutorial takes (which is a bit outdated and can be error-prone). If you're simply using input transforms (e.g., `Normalize`) within the model, you **don't** need to do anything to `X_baseline`.

**Comment by bmaxdk** [comment]
Transforming `X_baseline` is necessary if you manually transformed `train_X`. If BoTorch handles transformations internally, you don’t need to transform `X_baseline`. Always ensure consistency between `X_baseline` and the data format expected by your model.
